plugin/src/main/kotlin/com/trevjonez/composer/internal/ComposerBasePlugin.kt

```diff
@@ -100,6 +103,11 @@ abstract class ComposerBasePlugin<T> : Plugin<Project>
                                    ?.takeIf { it.isPresent }
                                    ?.also { composerTask.dependsOn(it) }
                                ?: getOutputDir(composerTask))
+
+    composerTask.extraApks.setFrom(variantConfigurator?.extraApks
+        ?.takeUnless{ it.isEmpty }
+        ?.also { composerTask.dependsOn(it) }
+        ?: extraApks)
```

Comment:
I think I should simple do `comoosertTask.extraApks.setFrom(extraApks)`

I did it like this because I was trying to be nice, but it's nit possible to specify buildType o flavor on AndroidTestUtil

Reply:
I think I see what you mean but I need to look at it in idea. github is too hard to parse right now for some reason.

Reply:
It works right, maybe it's overkill, we can decide to keep it that way being consistent with others or...
make it simpler and ignore buildVariants for AndroidTestUtil and test if it works or not